applications/LinearSolversApplication/custom_python/add_custom_solvers_to_python.cpp

```diff
@@ -231,6 +232,20 @@ void AddCustomSolversToPython(pybind11::module& m)
     register_solver<EigenPardisoLUSolver<complex>>(m, "ComplexPardisoLUSolver");
     register_solver<EigenPardisoLDLTSolver<complex>>(m, "ComplexPardisoLDLTSolver");
     register_solver<EigenPardisoLLTSolver<complex>>(m, "ComplexPardisoLLTSolver");
+
+    pybind11::class_<MKLILU0Solver<TUblasSparseSpace<double>,TUblasDenseSpace<double>>,
```

Comment:
Is it a solver or a preconditioner? if it is a solver ... which iterative solver is it?